utils/db_test.ts

```diff
@@ -71,7 +72,14 @@ Deno.test("[db] (get/create)Item()", async () => {
 
   assertEquals(await getItem(item.id), null);
 
+  const itemsCount =
+    (await getAnalyticsMetricPerDay("items_count", new Date()))?.valueOf() ??
+      0n;
```

Comment:
```suggestion
  assertEquals(await getAnalyticsMetricPerDay("items_count", new Date()), null);
```

Reply:
@iuioiua , can you pls explain this suggestion here?
I believe that if you do it like this, the assertion is going to fail. When it gets to this point, the itemsCount is probably equal to `2n` because of the two `createItem()` that were run on the test above (`" [db] getAllItems()"`)

We can't assume anything about the value o itemsCount, as we don't know how many tests will use it before in the code. We can only test that the itemsCount_after = itemsCount_before + 1.

Does it make sense? Did I understand correctly your suggestion?

Reply:
Perhaps, we should move the line that defines `itemsCount` elsewhere such that it doesn't require a default value. WDYT?